Sources/StytchCore/StytchClient/Events/StytchClient+Events.swift

```diff
@@ -0,0 +1,132 @@
+import Foundation
+public protocol EventsProtocol {
+    func logEvent(parameters: StytchClient.Events.Parameters) async throws
+}
+
+public extension StytchClient {
+    struct Events: EventsProtocol {
+        let router: NetworkingRouter<EventsRoute>
+
+        @Dependency(\.clientInfo) private var clientInfo
+        @Dependency(\.date) private var date
+        @Dependency(\.networkingClient) private var networkingClient
+        @Dependency(\.uuid) private var uuid
+        @Dependency(\.jsonEncoder) private var jsonEncoder
+
+        public func logEvent(parameters: Parameters) async throws {
+            let params = Params(
+                telemetry: .init(
+                    eventId: "event-id-\(uuid().uuidString)",
+                    appSessionId: "app-id-\(uuid().uuidString)",
```

Comment:
I believe this is something we want to create way earlier in the configuration flow. I know JS SDK generates these at calltime, like this, but there was a comment about wanting to hoist it so that we could track user flows across a static session. Take a look at what I did in the [Android implementation](https://github.com/stytchauth/stytch-android/blob/main/sdk/src/main/java/com/stytch/sdk/consumer/StytchClient.kt#L99), I tried to do that. The same comment can apply to `persistentId`, but I honestly have no clue what that's supposed to represent, haha, so kept it at calltime, like JS SDK.